invokeai/frontend/web/src/features/nodes/components/ProgressImageNode.tsx

```diff
@@ -1,18 +1,28 @@
 import { Flex, Image } from '@chakra-ui/react';
-import { NodeProps } from 'reactflow';
-import { InvocationValue } from '../types/types';
-
-import { useAppSelector } from 'app/store/storeHooks';
+import { NodeProps } from 'react-flow-renderer';
+import { useSelector, useDispatch } from 'react-redux';
+import { RootState } from 'app/store/store';
 import { IAINoContentFallback } from 'common/components/IAIImageFallback';
 import { memo } from 'react';
 import IAINodeHeader from './IAINode/IAINodeHeader';
-import IAINodeResizer from './IAINode/IAINodeResizer';
+import IAINodeResizer, { OnResize } from './IAINode/IAINodeResizer';
 import NodeWrapper from './NodeWrapper';
+import { nodeSizeChanged } from '../store/nodesSlice';
 
-const ProgressImageNode = (props: NodeProps<InvocationValue>) => {
-  const progressImage = useAppSelector((state) => state.system.progressImage);
+const ProgressImageNode = (props: NodeProps) => {
+  const progressImage = useSelector(
+    (state: RootState) => state.system.progressImage
+  );
+  const nodeSize = useSelector((state: RootState) => state.nodes.nodeSize);
+  const dispatch = useDispatch();
   const { selected } = props;
 
+  const handleResize: OnResize = (_, { width, height }) => {
+    const newSize =
+      width > height ? { width, height: width } : { width: height, height };
```

Comment:
Is this being done to keep it 1:1 ? Any reason for that?

Reply:
felt it looked more neat, that is really all

Reply:
Ah. I'll give it a shot and see. Ideally we want the user to be able to resize nodes anyway they want. But won't know until I try.

Reply:
@blessedcoolant I just removed the locked to 1:1 as it cause the above issue ⬆️